sdk/machinelearningservices/Azure.ResourceManager.MachineLearning/src/autorest.md

```diff
@@ -518,6 +518,9 @@ rename-mapping:
   StatusMessageLevel: JobStatusMessageLevel
   ImageType.azureml: AzureML
   ProtectionLevel: IntellectualProtectionLevel
+  OsPatchingStatus.latestPatchTime: -|date-time
+  OsPatchingStatus.scheduledRebootTime: -|date-time
```

Comment:
Consider renaming to `RebootScheduledOn`

Reply:
In generated code its ScheduledRebootOn
![image](https://github.com/user-attachments/assets/ae9a74f1-a8b2-49e7-9c8c-ca029e399ec9)